static/app/views/performance/newTraceDetails/traceGuards.tsx

```diff
@@ -94,10 +99,14 @@ export function isCollapsedNode(
   return node instanceof CollapsedNode;
 }
 
+export function isTraceError(value: TraceTree.NodeValue): value is TraceTree.TraceError {
+  return !!(value && 'level' in value && 'message' in value);
+}
```

Comment:
Seems like there's a similar function in `traceTreeNode.tsx` with the same name. Is that function still being used, and could we possibly consolidate?

Reply:
This leads to a circular dependency. Hence we see a number of duplicate guards in `traceTreeNode.tsx` I can try solving this in a followup pr.